tidb-cloud/set-up-vpc-peering-connections.md

```diff
@@ -36,20 +36,18 @@ You can set the CIDR when creating the first TiDB Dedicated cluster. If you want
 
     > **Note:**
     >
-    > - To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field. 
+    > - To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field.
     > - For AWS Region, it is recommended to configure an IP range size between `/16` and `/23`. Supported network addresses include:
-    >     - 10.250.0.0 - 10.251.255.255 
+    >     - 10.250.0.0 - 10.251.255.255
     >     - 172.16.0.0 - 172.31.255.255
     >     - 192.168.0.0 - 192.168.255.255
-
     > - For Google Cloud Region, it is recommended to configure an IP range size between `/19` and `/20`. If you want to configure an IP range size between `/16` and `/18`, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md). Supported network addresses include:
     >     - 10.250.0.0 - 10.251.255.255
     >     - 172.16.0.0 - 172.17.255.255
     >     - 172.30.0.0 - 172.31.255.255
-
     > - TiDB Cloud limits the number of TiDB Cloud nodes in a region of a project based on the CIDR block size of the region.
```

Comment:
FYI. My change to the preceding lines in the **Note** is to fix the display on the site.
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/573ba059-2df8-42ba-89cf-6ea180a8672a">